solar-land/scss/_variables.scss

```diff
@@ -0,0 +1,12 @@
+/*------------------------------------*\
+  #COLORS
+\*------------------------------------*/
+$primaryColor: #0dd47e;
+$darkTextColor: #1e1e1e;
+$secondaryTextColor: #8d959d;
+$backgroundLightColor: #f2f4f4;
+
+// defining fonts
+@import url("https://fonts.googleapis.com/css2?family=Quicksand:wght@300..700&display=swap");
```

Comment:
I think it is better to have importing the font in typography file instead of variables. What do you think?

Reply:
because i should add a variable for font i added it into this file.
typography is for creating classes etc...